src/structures/suggestion.ts

```diff
@@ -309,11 +314,11 @@ export class Suggestion {
 
 		if (!message.hasThread)
 			await message.startThread({
-				name: `Suggestion: ${title}`,
+				name: messageOptions.embeds[0].data.title ?? "Suggestion",
```

Comment:
Maybe use a truncated form of the description for thread naming? It would be kind of confusing if all threads are named the same.

Reply:
Good catch, I'll truncate the description for the thread name.